app/services/requests_total_items_service.rb

```diff
@@ -0,0 +1,41 @@
+class RequestsTotalItemsService
+  def initialize(requests:)
+    @requests = requests
+  end
+
+  def calculate
+    return unless requests
+
+    request_items_array = []
+    items_names = Item.where(id: request_items_ids).as_json(only: [:id, :name])
+
+    request_items.each do |items|
+      items.map do |json|
```

Comment:
Considering we're not returning the map result to serve other purposes could it be just an `each`?